src/analysis/functions.rs

```diff
@@ -238,51 +260,201 @@ fn analyze_function(
     );
     commented |= ret.commented;
 
+    let mut params = func.parameters.clone();
     let mut parameters = function_parameters::analyze(
         env,
-        &func.parameters,
+        &params,
         configured_functions,
         disable_length_detect,
         async,
-        in_trait
+        in_trait,
     );
     parameters.analyze_return(env, &ret.parameter);
 
+    if let Some(ref f) = ret.parameter {
+        if let Type::Function(_) = env.library.type_(f.typ) {
+            if env.config.work_mode.is_normal() {
+                warn!("Function \"{}\" returns callback", func.name);
```

Comment:
Fancy, this exists? Can you give an example? :)

Reply:
Yes it does. I don't have one right now but if you want to see want, just comment this check and you'll see some magic. :)

Reply:
gio::get_poll_func, glib::get_poll_func, gtk::lookup_callback_symbol, pangocairo::context_get_shape_renderer

Reply:
Thanks @EPashkin! :)